doc/reference/rpc_command.rst

```diff
@@ -0,0 +1,1227 @@
+Legacy RPC Commands
+===================
+
+
+.. This document is generated via the `development:generate-documentation` command
+
+
+.. contents::
+   :depth: 2
+   :backlinks: none
+   :local:
+
+
+.. _RpcHandler_StatusHandler:
+
+
+_RpcHandler_StatusHandler
+-------------------------
+
+
+.. _RpcCommand_status_type:
```

Comment:
not sure why this is RpcCommand but it's for `_RpcHandler_`, should probably have been `RpcHandler_StatusHandler_Param_status_typ` or something (or simply remove the reference declaration), but it's fine :+1: